python/pyspark/ml/tuning.py

```diff
@@ -106,7 +107,7 @@ class CrossValidator(Estimator):
     >>> lr = LogisticRegression()
     >>> grid = ParamGridBuilder().addGrid(lr.maxIter, [0, 1]).build()
     >>> evaluator = BinaryClassificationEvaluator()
-    >>> cv = CrossValidator(estimator=lr, estimatorParamMaps=grid, evaluator=evaluator)
+    >>> cv = CrossValidator(estimator=lr, estimatorParamMaps=grid, evaluator=evaluator, seed=42)
```

Comment:
This is just an example though.  But I agree we could remove the seed now to simplify the example.  I'll do that.